### DISCUSSION TITLE
how to override master app and dashboard template

### DISCUSSION THREAD

**Discussion by pratikraman-dev** [orchidsoftware/platform#2309] [Q&A]
i want to use admin lte dashboard so im trying to make changes in master app template and dashboard template but cannot override those still  i have to make changes in vendor folder view files i haved copied all thing in resource but changes are getting reflected 

![image](https://user-images.githubusercontent.com/8825570/180179404-1a33c4be-6505-4dbc-830e-9d29ea7fafb1.png)

**Comment by kaizirlewagen** [comment]
Can this be a solution?

https://orchid.software/en/docs/configuration/#overriding-blade-templates

**Comment by astersnake** [comment]
I don't understand your question very well, but if you are trying to change the blade templates, you should follow the link by @kaizirlewagen. Also, you should not modify the vendor folder. If you try to modify the CSS or javascript, you will see no effect on your application because these files must be compiled.
